fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadInfo.java

```diff
@@ -457,7 +457,11 @@ private void setOptionalFromTSLPutRequest(TStreamLoadPutRequest request, Databas
         }
 
         if (request.isSetCompression_type()) {
-            payloadCompressionType = CompressionUtils.findTCompressionByName(request.getCompression_type());
+            TCompressionType compression = CompressionUtils.findTCompressionByName(request.getCompression_type());
+            if (compression == null) {
+                throw new UserException("unsupported compression type: " + request.getCompression_type());
+            }
+            payloadCompressionType = compression;
```

Comment:
Why not directly use ` payloadCompressionType = CompressionUtils.xxx` rather than using a new variable `compression`?